src/main/groovy/org/assertj/generator/gradle/internal/tasks/AssertionsGeneratorReport.groovy

```diff
@@ -17,24 +17,25 @@ import org.assertj.assertions.generator.AssertionsEntryPointType
 
 import static com.google.common.collect.Maps.newTreeMap
 import static com.google.common.collect.Sets.newTreeSet
-import static org.apache.commons.lang3.ArrayUtils.isNotEmpty
 import static org.apache.commons.lang3.StringUtils.remove
 import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace
 
 class AssertionsGeneratorReport {
+    File directoryPathWhereAssertionFilesAreGenerated
+    Collection<String> inputPackages
+    Collection<String> inputClasses
+    Exception exception
+    Collection<TypeToken<?>> excludedClassesFromAssertionGeneration
+    final List<String> userTemplates
+    final Set<String> inputClassesNotFound
 
     private static final String INDENT = "- "
     private static final String SECTION_START = "--- "
     private static final String SECTION_END = " ---\n"
-    private File directoryPathWhereAssertionFilesAreGenerated
-    private Set<String> generatedCustomAssertionFileNames
-    private Map<AssertionsEntryPointType, File> assertionsEntryPointFilesByType
-    private Collection<String> inputPackages
-    private Collection<String> inputClasses
-    private Exception exception
-    private Collection<TypeToken<?>> excludedClassesFromAssertionGeneration
-    private Set<String> inputClassesNotFound
-    private List<String> userTemplates
+    private static final String Line = System.lineSeparator()
```

Comment:
this makes the `append(Line)` easier to read than `append(System.lineSeparator())`